raphael-sim/src/actions.rs

```diff
@@ -870,3 +903,41 @@ impl Action {
         }
     }
 }
+
+impl  Action {
```

Comment:
There's already an `impl Action` block right above. No need to create a new block.